### ISSUE TITLE
[DX] Does system_get_info need to be updated to include 'layout' as $type?

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2384]
https://api.backdropcms.org/api/backdrop/core!modules!system!system.module/function/system_get_info/1

> Returns an array of information about enabled modules or themes.

> This function returns the information from the {system} table corresponding to the cached contents of the .info file for each active module or theme.

>Parameters

> **$type**: Either 'module' or 'theme'.

PS: perhaps we also need a `project_get_info` helper function (is that what they're called?).

**Comment by BWPanda** [comment]
If we did, looks like we'd need to update `[system_list](https://docs.backdropcms.org/api/backdrop/core%21includes%21module.inc/function/system_list/1)`. But I'm wondering, are layouts stored in the DB at all...?

**Comment by indigoxela** [comment]
> But I'm wondering, are layouts stored in the DB at all...?

No, only caching, which isn't relevant here. Layouts are config.

What would be the benefit of adding layouts to system_get_info?

**Comment by klonos** [comment]
@indigoxela and @BWPanda you are confusing "layouts" (which are config) with "layout templates" (which are a type of project, same as modules and themes). I know that the confusion stems from the use of `'layout'` in the issue title, but what I am referring to is the `type = layout` property in the .info files of layout templates.

This is not your fault - we have made a naming poopoo when we introduced layouts and layout templates in Backdrop, which is now hard to disambiguate 😓  ...see: #5341 and other similar issues/discussions.
